lib/improver/wxcode/weather_symbols.py

```diff
@@ -73,8 +74,10 @@ def __init__(self, wxtree='high_resolution'):
         self.wxtree = wxtree
         if wxtree == 'global':
             self.queries = wxcode_decision_tree_global()
+            self.start_node = START_NODE_GLOBAL
```

Comment:
I don't think we need to have a different name for the decision tree (and the start node) in each file - the files themselves have different names - that should be sufficient.

Also we can do the `import` here, so we only import the tree that we need.

Reply:
I agree this would be neater but I think it is outside the remit of this change and would probably need quite a few changes to  unit tests so I will raise another ticket to do this at a later date.

Reply:
No problem, that's fine with me.